src/utils/getFiltered.ts

```diff
@@ -0,0 +1,64 @@
+import { Person } from '../types';
+import { Sort } from '../types/Sort';
+
+export function getFilteredPeople(
+  people: Person[],
+  {
+    sort,
+    order,
+    query,
+    sex,
+    centuries,
+  }: {
+    sort: string | null;
+    order: string | null;
+    query: string;
+    sex: string;
+    centuries: string[];
+  },
+) {
+  let filteredPeople = [...people];
+  const normilizeQuery = query.toLowerCase();
+
+  if (sort) {
+    filteredPeople.sort((p1, p2) => {
+      switch (sort) {
+        case Sort.Name:
+        case Sort.Sex:
+          return p1[sort].localeCompare(p2[sort]);
+        case Sort.Born:
+        case Sort.Died:
+          return p1[sort] - p2[sort];
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (order) {
+    filteredPeople.reverse();
+  }
```

Comment:
Reversing the array after sorting it is not the best approach for handling the 'order' parameter. It would be more efficient to handle the 'order' in the sorting function itself. The current approach could lead to performance issues if the array is large.